source/distributed/tria.cc

```diff
@@ -567,6 +569,622 @@ namespace
   }
 
 
+#  ifdef DEAL_II_WITH_P4EST_SEARCH_LOCAL
+  template <int dim>
+  class PartitionSearch
+  {
+  public:
+    PartitionSearch()
+    {}
```

Comment:
Just another sanity check.
```suggestion
    {
      Assert(dim > 1, ExcNotImplemented());
    }
```